### PR TITLE
Refactor enclave, manager and utilities

### PR DESCRIPTION
**Pull request by manju956**
Includes fixes for following
- Rename work_load_id to workload_id
- Fix incorrect json error message when encryptedRequestHash field is missing in request
- Add return values for functions decrypt_data and decrypted_response
- Provide proper hex string values for ApplicationTypeId and OrganizationId
- worker_id is made valid hex string

Signed-off-by: manju956 <manjunath.a.c@intel.com>

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ version: '3.5'
 
 services:
   tcf:
-    container_name: tcf
+    container_name: tcf-manju
```
Comment (danintel) [inline docker-compose.yaml]: I suggest restoring the name `tcf` or using another name than` tcf-manju`. Was this for debugging?

File: config/tcs_config.toml [hunk 2]
```diff
@@ -66,8 +66,8 @@ WorkOrderSyncUri = "http://localhost:8080"
 WorkOrderAsyncUri = "http://localhost:8080"
 WorkOrderPullUri = "http://localhost:8080"
 WorkOrderNotifyUri = "http://localhost:8080"
-ApplicationTypeId = "Application_01"
```
Comment (Ram-srini) [inline config/tcs_config.toml]: Instead adding hard coded random value, can you add something like in the python code.
"Application_01".encode("UTF-8").hex()
"Organization_01".encode("UTF-8").hex()

File: docker-compose.yaml (hunk 1 above)
Comment (manju956) [reply docker-compose.yaml]: this file was mistakenly added in the commit list. I will remove it.

File: config/tcs_config.toml (hunk 2 above)
Comment (manju956) [reply config/tcs_config.toml]: If enclave manager does the hex conversion, aren't we modifying client specific config?
As per spec, users need to input their applicationtypeId and OrganizationId in hex format. I will add this comment.

**Pull request merged**